scripts/main/build.js

```diff
@@ -128,29 +116,83 @@ build.album = function(data, disabled = false) {
 
 };
 
-build.photo = function(data) {
+build.photo = function (data) {
 
 	let html = '';
-	let { path: retinaThumbUrl } = lychee.retinize(data.thumbUrl);
+	let thumbnail = '';
+	let thumb2x = '';
+
+	let isVideo = data.type && data.type.indexOf('video') > -1;
+	if (data.thumb === 'uploads/thumb/' && isVideo) {
+		thumbnail = `<span class="thumbimg"><img src='dist/play-icon.png' alt='Photo thumbnail' data-overlay='false' draggable='false'></span>`
+	} else if (lychee.layout === '0') {
+
+		if (data.thumb2x) { // Lychee v4
+			thumb2x = data.thumb2x
+		} else { // Lychee v3
+			let {path: thumb2x} = lychee.retinize(data.thumbUrl)
+		}
+
+		if (thumb2x !== '') {
+			thumb2x = `srcset='${thumb2x} 2x'`
+		}
+
+		thumbnail = `<span class="thumbimg${isVideo ? ' video' : ''}">`;
```

Comment:
There are a few places here where it checks for videos to add the video class, but doesn't add the play icon. Am I missing something, or should this be all-or-nothing?

Reply:
it should add the play icon if the video thumbnail is not available.

Reply:
Ah, of course!